readme.md

```diff
@@ -32,12 +32,18 @@ Source object to copy properties from.
 
 #### mapper
 
-Type: `Function`
+Type: `(sourceKey, sourceValue, source) => [targetKey, targetValue, mapperOptions?]`
```

Comment:
TIL, a question mark after an item in a tuple will make the item optional. https://www.typescriptlang.org/docs/handbook/2/objects.html#tuple-types

If this change requires a newer version of TypeScript, we may as well use [named tuple elements](https://github.com/Microsoft/TypeScript/issues/28259#issuecomment-452912594).

Reply:
> If this change requires a newer version of TypeScript, we may as well use named tuple elements.

👍